### ISSUE TITLE
[Next] Behavior when multiple configs on page (Show Warning?)

### ISSUE THREAD

**Issue by JeffersGlass** [pyscript/pyscript#1745]
I think we can improve our UX when the user (mistakenly) has multiple configs on the page.

From experimentation and what I understand from [this section of config.js](https://github.com/pyscript/pyscript/blob/b4c686f411f1484852690478885508fd1fd56674/pyscript.core/src/config.js#L54-L65), the precedence for where to find the config on the page is:

1. The first `<py-config>` tag, if any
2. The first `<py-script>` or `<script type="py">` tag with a `config` attribute that `doesn't` have a worker attribute
3. All other configs (if any) are ignored

This is good I think. But I think it would be good to show a warning if the user does have multiple configs on the page. Currently, if a new user does:
```py
<script type="py" config='{"packages":["rich"]}'>
    import rich
    print(dir(rich))
</script>
<script type="py" config='{"packages":["requests"]}'>
    import requests
    print(dir(requests))
</script>
```
The error they get is "No module called requests", when we might as well show a warning that we're ignoring their second config.

Thoughts?

**Comment by WebReflection** [comment]
I am not saying this is not a valid issue, I am saying how it works (due reasons!):

  * we can't have multiple config on the page for the main thread, just like we couldn't before on "*classic*" *PyScript*
  * the reason for previous point is that one can't enable or disable plugins at runtime, or change files to merge, as these are all operations needed in the first place to make *PyScript* work ... this is nothing new though ...
  * the `worker` is a whole new story and it has its own whole new story behavior ... a worker is inevitably a new thread with a new "*pyscript*" interpreter and because of that, it can have any config it likes, that's the whole purpose of having a worker, when more files to fetch or more packages to load **must** be allowed as those are non-blocking anyway ... so `worker` attribute **must** allow a custom config *but* it inherits one from the main thread, when that's meant to describe what a worker should do (scenario: there's only a worker *PyScript* element on the page, it won't ever block, but there is also a `<py-config>` tag, which is what lands by default in such worker)

**Comment by JeffersGlass** [comment]
Good reasons all! They make good sense, and I think one-page-one-config makes total sense. My only contribution here really was to suggest showing a warning if the user tries (mistakenly) to use more than one config. 

If that seems like a reasonable add, I can submit a PR? Not a blocker for the release in any way, just a nice-to-have in my opinion.

**Comment by WebReflection** [comment]
I think @antocuni would strongly agree with you on this, all I am saying is that:

> one-page-one-config

nope, **one thread, one config** is what we can offer now.

**Comment by WebReflection** [comment]
OK, I think there is a fundamental question to answer here before any MR lands (I was quickly checking what I had to do in order to show something meaningful) ... to me, if the config is from an attribute I'd expect other scripts or tags to use such config as if it's the only one needed ... example

```html
<script type="mpy" config="my.py">print(1)</script>
<script type="mpy">print(2)</script>
```

I would expect this example to be identical to:

```html
<mpy-config>
[[fetch]]
files = ["a.py"]
</mpy-config>
<script type="mpy">print(1)</script>
<script type="mpy">print(2)</script>
```

meaning, all scripts would follow the single config file defined in the page ... yes?

To extend the use case, I would also consider the following equivalent, as all configs point to the same file:

```html
<script type="mpy" config="my.py">print(1)</script>
<script type="mpy" config="my.py">print(2)</script>
```

In short, I am saying that if there are multiple `<mpy-config>` or multiple `<py-config>` as tags on the page, we throw an error (but one single `<mpy-config>` plus one single `<py-config>` would be just fine) but when it comes to attributes, I would throw only if the `config` points at different files per each *type* we handle ... deal?
